src/app/views/provisioning/summary.html

```diff
@@ -9,19 +9,42 @@
       </div>
     </mno-widget-header>
     <mno-widget-body class="no-padding">
+      <table class="table table-striped">
+        <thead>
+        <tr>
+          <th id="order-type">{{ 'mnoe_admin_panel.dashboard.provisioning.summary.order_type' | translate }}</th>
+          <th ng-show="vm.subscription.product.product_type == 'application'">{{ 'mnoe_admin_panel.dashboard.provisioning.summary.pricing_info.title' | translate }}</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <td>{{ vm.orderTypeText | translate }}</td>
+          <td ng-show="vm.subscription.product.product_type == 'application'">
+            <span ng-hide="vm.singleBilling">
+              {{ 'mnoe_admin_panel.dashboard.provisioning.summary.pricing_info.single_billing_disabled' | translate }}
```

Comment:
Rather than swapping between span, would it be easier to have the controller returning the locale key?

Reply:
sure